src/app/views/record/record.html

```diff
@@ -6,9 +6,9 @@ <h2>{{$ctrl.record.entity}} {{$ctrl.getIdentifier($ctrl.record)}}</h2>
 
   <div class="main-details" layout="row" layout-align="center center">
     <div ng-if="$ctrl.record.img1024" flex="100">
-      <img ng-src="{{$ctrl.record.img1024}}" href="{{$ctrl.record.originalUrl}}" />
+      <a href="{{$ctrl.record.img1024}}"><img ng-src="{{$ctrl.record.img512}}"></a>
```

Comment:
can you change the href to `{{ $ctrl.record.originalUrl || $ctrl.record.img1024 }}` so we link to the original if present

Reply:
Since the original could be very large (50Mb or greater), i would rather keep the link to the original photo in the recordDetails and have the user explicitly choose this...